object/types.proto

```diff
@@ -222,4 +228,9 @@ message SplitInfo {
   // split header with the original object header and a sorted list of
   // object parts.
   neo.fs.v2.refs.ObjectID link = 3;
+
+  // The identifier of the initial object for split hierarchy parts. It contains
+  // zero payload and the original object (also known as a "parent") header in the
```

Comment:
known by who? all other fields and `SplitInfo` itself use `original object` term, and it's enough imo

Reply:
> known by who?

by an api reader only a few lines [above](https://github.com/nspcc-dev/neofs-api/blob/8eb21aac3f6a811016536751a1b6c4d8f02c0428/object/types.proto#L165). again, API docs are not perfect so more words are better imo